src/Service/Loan.php

```diff
@@ -97,7 +97,10 @@ public function create(LoanEntity $loan, $paymentDate)
      */
     private function createInstallment($loan, $price, $paymentDate)
     {
-        $toBePaidStatus = $this->installmentStatusRepository->getToReceive();
+        $today = new \DateTime(date('Y-m-d'));
+        $today > $paymentDate ?
```

Comment:
I added this condition to installments that are created with payment date prior to current.